ext/bulk_add/theme.php

```diff
@@ -12,9 +12,9 @@ public function display_upload_results(Page $page) {
 		$page->add_block(new NavBlock());
 		$html = "";
 		foreach($this->messages as $block) {
-			$html .= "<br/>" . html_escape($html);
+			$html .= "<br/>" . $block->body;
```

Comment:
html_escapeing at this location causes the HTML being sent as part of the error messages to get messed up. Fortunately the parts that generate the error message html_escape the paths when they create the messages, so I think this is okay.